cuda_profiler.conf

```diff
@@ -0,0 +1,19 @@
+method
+gputime
+cputime
+occupancy
+gpustarttimestamp
+gpuendtimestamp
+gridsize3d
+threadblocksize
+dynsmemperblock
+stasmemperblock
+regperthread
+memtransferdir
+memtransfersize
+memtransferhostmemtype
+streamid
+cacheconfigrequested
+cacheconfigexecuted
+countermodeaggregate
+conckerneltrace
```

Comment:
Can this file move into platform/cuda or topo/gpu so that non-GPU users don't see it?